entity-service/src/main/java/org/hypertrace/entity/service/EntityServiceConfig.java

```diff
@@ -14,8 +14,6 @@
   private final Config entityServiceConfig;
 
   public EntityServiceConfig(Config config) {
-
-    LOG.info(config.toString());
```

Comment:
This is anyway not required as we are already logging appConfig in the base framework - https://github.com/hypertrace/service-framework/blob/d3370d2309d95bc8fe7ca7275f66f351454422a6/platform-service-framework/src/main/java/org/hypertrace/core/serviceframework/PlatformService.java#L97

Reply:
Won't the credentials be printed in the app config then?

Reply:
Yes. And as discussed here - https://github.com/hypertrace/hypertrace/issues/137, I am going to apply option (a) even at the framework level too. The consumer should print requires things as per need with a meaningful message to make it actionable.  So, this is anyway needed.

Reply:
This is the PR for the same - https://github.com/hypertrace/service-framework/pull/22, once that is merged, I have to upgrade appropriate services, at least attribute, entity and config-bootstrapper for now for postgres sensitive information.